docs/contents/components/forms/dropzone/index.ja.mdx

```diff
@@ -219,10 +249,24 @@ import {
     </DropzoneIdle>
 
     <VStack gap="2xs">
-      <Text fontSize="xl">Drag images here or click to select files</Text>
-      <Text fontSize="sm">
-        Attach as many files as you like, each file should not exceed 3mb
-      </Text>
+      <For
```

Comment:
There is no need to use `For` here. I feel that the modification makes it less readable.

Reply:
Fixe.